sorc/ncep_post.fd/AVIATION.f

```diff
@@ -346,7 +353,13 @@ SUBROUTINE CALCAT(U,V,H,U_OLD,V_OLD,H_OLD,CAT)
 !cvg=-(du/dx+dv/dy)
            CVG = -((0.5*(U(I,J)+U(I,J-1))-0.5*(U(I-1,J)+U(I-1,J-1)))*10000./DX(I,J) &
                 +(0.5*(V(I,J)+V(I-1,J))-0.5*(V(I,J-1)+V(I-1,J-1)))*10000./DY(I,J))	   
+           ELSE
+            DEF = SPVAL
```

Comment:
Line 357, should it be DSH = SPVAL ?
@JesseMeng-NOAA

Reply:
line 351  DEF = SQRT (DSH*DSH + DST*DST)
DEF and CVG are the final computed variables from this block.